libraries/c_sdk/aws/shadow/test/system/aws_iot_tests_shadow_system.c

```diff
@@ -422,6 +429,28 @@ static void _updateGetDeleteBlocking( IotMqttQos_t qos )
 
 /*-----------------------------------------------------------*/
 
+/**
+ * @brief Establish an MQTT connection. Retry if enabled.
+ */
+static IotMqttError_t _mqttConnect( const IotMqttNetworkInfo_t * pNetworkInfo,
+                                    const IotMqttConnectInfo_t * pConnectInfo,
+                                    uint32_t timeoutMs,
+                                    IotMqttConnection_t * const pMqttConnection )
+{
+    IotMqttError_t status = IOT_MQTT_STATUS_PENDING;
+
+    /* AWS IoT Service limits only allow 1 connection per MQTT client ID per second.
+     * Wait until 1100 ms have elapsed since the last connection. */
+    uint32_t periodMs = 1100;
```

Comment:
Should this be a configurable macro similar to the retry count?

Reply:
I would prefer that, but I noticed that _mqttConnect in the MQTT system tests had it as a local var.

Reply:
Okay Sarena agreed to make this a macro and I'm defining it in the iot_config_common.h file in a separate PR